tools/clang/test/CMakeLists.txt

```diff
@@ -90,6 +95,7 @@ add_lit_testsuite(check-clang "Running the Clang regression tests"
   #LIT ${LLVM_LIT}
   PARAMS ${CLANG_TEST_PARAMS}
     skip_taef_exec=True
+    skip_hash_stability=True
```

Comment:
Hash stability should be enabled by default, and we shouldn't have options to skip it:

https://github.com/microsoft/DirectXShaderCompiler/pull/5548

Reply:
Fixed.